scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementController.java

```diff
@@ -0,0 +1,203 @@
+
+      sceneRelatedPoint = new Point(-4.4610562, 48.3902727, 70, SpatialReferences.getWgs84());
+
+      // create a red triangle symbol
+      SimpleMarkerSymbol triangleSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.TRIANGLE, 0xFFFF0000, 12);
+
+      // create a text symbol for each surface placement type
+      TextSymbol drapedBillboardedText =
+          new TextSymbol(14, "DRAPED BILLBOARDED", 0xFF0000FF, TextSymbol.HorizontalAlignment.LEFT, TextSymbol.VerticalAlignment.MIDDLE);
+      drapedBillboardedText.setOffsetX(20);
+
+      TextSymbol drapedFlatText =
+          new TextSymbol(14, "DRAPED FLAT", 0xFF0000FF, TextSymbol.HorizontalAlignment.LEFT, TextSymbol.VerticalAlignment.MIDDLE);
+      drapedFlatText.setOffsetX(20);
+
+      TextSymbol relativeText =
+          new TextSymbol(14, "RELATIVE", 0xFF0000FF, TextSymbol.HorizontalAlignment.LEFT, TextSymbol.VerticalAlignment.MIDDLE);
+      relativeText.setOffsetX(20);
+
+      TextSymbol absoluteText =
+          new TextSymbol(14, "ABSOLUTE", 0xFF0000FF, TextSymbol.HorizontalAlignment.LEFT, TextSymbol.VerticalAlignment.MIDDLE);
+      absoluteText.setOffsetX(20);
+
+      TextSymbol relativeToSceneText =
+        new TextSymbol(14, "RELATIVE TO SCENE", 0xFF0000FF, TextSymbol.HorizontalAlignment.RIGHT, TextSymbol.VerticalAlignment.MIDDLE);
+      relativeToSceneText.setOffsetX(-20);
+
+      // add the point graphic and text graphic to the corresponding graphics overlay
+      drapedBillboardedOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, triangleSymbol));
+      drapedBillboardedOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, drapedBillboardedText));
+
+      drapedFlatOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, triangleSymbol));
+      drapedFlatOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, drapedFlatText));
+
+      relativeOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, triangleSymbol));
+      relativeOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, relativeText));
+
+      absoluteOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, triangleSymbol));
+      absoluteOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, absoluteText));
+
+      relativeToSceneOverlay.getGraphics().add(new Graphic(sceneRelatedPoint, triangleSymbol));
+      relativeToSceneOverlay.getGraphics().add(new Graphic(sceneRelatedPoint, relativeToSceneText));
+
+      // add graphics overlays to the scene view
+      sceneView.getGraphicsOverlays().addAll(Arrays.asList(drapedBillboardedOverlay, relativeOverlay, absoluteOverlay, relativeToSceneOverlay));
+
+      // link up the ToggleGroup RadioButtons to the corresponding draped overlay
+      drapedBillboardedRadioButton.setUserData(drapedBillboardedOverlay);
+      drapedFlatRadioButton.setUserData(drapedFlatOverlay);
+
+      // listener for toggles between billboarded and flat draped surface placement modes on the JavaFX ToggleGroup
```

Comment:
Maybe we can rephrase this comment a bit to describe the action that the code performs:
```suggestion
      // add a listener to the toggle group to switch between billboarded and flat draped surface placement modes
```